docs/userstylesyml.md

```diff
@@ -0,0 +1,149 @@
+     url: https://github.com/<github-username>
+   ```
+
+2. Locate the `userstyles` section and add your port to the list.
+
+   You should make changes to everything wrapped in `<>` and remove the `<>`
+   characters. If you need help with any of the fields, you should refer to
+   other ports in the `userstyle.yml` file. If you would like extra guidance,
+   you can ask for help in the pull request or join our
+   [Discord](https://discord.com/servers/catppuccin-907385605422448742).
+
+   `Required Fields:`
+
+   ```yaml
+   <port>:
+     name: <name>
+     # Up to 3 categories are allowed, and having at least one is required.
+     # The first category is considered the "primary" category and impacts where the userstyle appears on the README.
+     categories: [<primary-category>]
+     color: <color>
+     readme:
+       app-link: <url>
+       current-maintainers: [*<github-username>]
+   ```
+
+   `All Fields:`
+
+   These extra `usage` & `faq` keys will ensure that you can add important
+   information about the port to the README.
+
+   The `icon` key is best explained the schema specification itself.
+
+   The `past-maintainers` key is a list of people who have maintained the port
+   in the past. We encourage all maintainers to add/remove themselves from this
+   list as they see fit.
+
+   Remember that these 5 fields are **optional**.
+
+   ```yaml
+   <port>:
+     name: <name>
+     categories: [<primary-category>, <optional-secondary-category>, <optional-third-category>]
+     color: <color>
+     icon: <icon>                                    # OPTIONAL
+     readme:
+       app-link: "<url>"
+       usage: |+                                     # OPTIONAL
+         > [!NOTE]
+         > <your notes>
+       faq:                                          # OPTIONAL
+         - question: <question>
+           answer: <answer>
+       current-maintainers: [*<github-username>]
+       past-maintainers: [*<github-username>]        # OPTIONAL
+   ```
+
+### Adding yourself as a maintainer
+
+To add yourself as a maintainer, you need to add your GitHub username to the
+`maintainers` array in the `userstyles.yml` file. This file is located in the
+`scripts` directory. The `maintainers` array, is an array of objects, each
+object should contain a `name` and a `url` key. And optionally a `name` key,
+which is your preferred name.
+
+```yaml
+maintainers:
+  - &your-username
+    name: Your Preferred Name
+    url: https://github.com/your-username
+```
+
+Then you need to add your username to the `current-maintainers` array in for the
+userstyle you want to maintain, in this case `youtube`.
+
+```yaml
+userstyles:
+  youtube:
+    name: YouTube
+    categories: [entertainment, social_networking, photo_and_video]
+    icon: youtube
+    color: red
+    readme:
+      app-link: "https://youtube.com"
+      current-maintainers: [*isabelroses *your-username]
+      past-maintainers: [*elkrien]
+```
+
```

Comment:
I think we should add some information on what happens after they get added as a maintainer. E.g. They'll be added to the `userstyle-maintainers` GitHub team.

Also some information on what is expected of them as a maintainer?

Reply:
I'm unsure what to add about what is expected of a maintainer.

Reply:
- Responsibility to merge PRs that they maintain for
- Code of conduct and being respectful to other maintainers and contributors
- Keeping theme up to date (totally fine if they can't anymore, but they should remove themselves from the list so it is marked as possibly not fully working)